tests/chaos/utils.py

```
@@ -388,23 +378,65 @@ def rebooting_node(node, utility_pods):
+        ResourceNotFoundError:
+            If no matching workloads are found in the given namespace.
     """
-    resource_objs = get_resources_by_name_prefix(
-        prefix=pod_prefix,
-        namespace=namespace,
-        api_resource_name=resource,
-    )
-    if not resource_objs:
-        raise ResourceNotFoundError(f"No resources found with prefix {pod_prefix} in namespace {namespace}")
+    if not isinstance(resources, list):
+        raise TypeError("resources must be a list of Deployment or DaemonSet")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)